### ISSUE TITLE
Question components - my answer/model answer button accessibility

### ISSUE THREAD

**Issue by moloko** [adaptlearning/adapt_framework#911]
The model answer/my answer button is redundant for those using a screen reader as there is no indication which options have ticks/crosses.

Represent model+my answer via an API on questionView for each component. Buttons view to change aria labels on model answer button to represent component model/my answer state.

@oliverfoster to split into separate subtasks

moved from [ABU-1072](https://adaptlearning.atlassian.net/browse/ABU-1072)

**Comment by oliverfoster** [comment]
This behaviour has a few issues I'd like to document here.

This behaviour is not particularly intuitive to accessible and non-accessible users. The user experience for sighted users created by making a large change in the presented UI data (each item is remarked with ticks/crosses) and the ability to toggle the two modes in quick succession. For non-sighted users asking them to understand this large data shift in the UI data by tabbing through the individual items is asking a huge amount of cognitive effort for very little reward.

Bearing in mind that we would ideally not have two distinct modes of use, accessibility mode and non-accessibility mode, that Adapt should present a single version to both user groups, any differentiating in behaviour here should be avoided.

I would recommend removing this behaviour altogether and attempting to provide a better / more generated method of generating itemized feedback.

**Comment by oliverfoster** [comment]
Having thought about this for a while I think I've got a solution to this issue.

Issue:
* User item answers and correct item answers should be represented separately
* Feedback should be inline
* Correct item answers should be inline

Solutions:
* Remove/disable my/model answer feature from core
* Change tutor to inline feedback
* Add correct item answer representation views to components and allow tutor to render under the inline feedback

![20180522_132852](https://user-images.githubusercontent.com/7974663/40362761-617b14b6-5dc5-11e8-8491-4b52b867b4ee.jpg)
![20180522_132713](https://user-images.githubusercontent.com/7974663/40362766-636fd806-5dc5-11e8-9fa3-91acdb82c803.jpg)

**Comment by oliverfoster** [comment]
easier solution in #2241
